NinchatSDKSwift/Implementations/Managers/Session Manager/NINChatSessionManager.swift

```diff
@@ -78,6 +78,9 @@ protocol NANChatSessionMessenger {
     
     /** Load channel history. */
     func loadHistory(completion: @escaping CompletionWithError) throws
+
```

Comment:
NANChatSessionMessenger... that's a typo, right? :smile:

Reply:
Oh, yes, it is a typo. 🤦‍♂️